scripts/run_python_test.sh

```diff
@@ -18,9 +18,9 @@ WORKSPACE=${SCRIPT_DIR}/..
 
 unset PYTHONPATH
 
-cd $WORKSPACE
+cd "$WORKSPACE"
```

Comment:
Is `"$WORKSPACE"` the same as `${WORKSPACE}`?

Reply:
No, the quotes are intentional. https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
I suggest you use this tool called `shellcheck` along with `pre-commit`.